00_Project/sos/SERVICES/sos/sos_config.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * sos_config.c
+ *
+ * Created: 5/29/2023 8:37:19 PM
+ *  Author: sharpel
+ */ 
+
+
+#include "../../COMMON/std_types.h"
+#include "sos_config.h"
+#include "../../MCAL/timer/timer_interface.h"
```

Comment:
Remove any path information from your includes. Any path information shall be defined in the project to avoid touching the source code when you change any header file location

Reply:
Done